packages/sdk/src/react/hooks.ts

```
@@ -89,205 +95,178 @@ export function useEntityId(...keys: BigNumberish[]): BigNumberish {
 
-        if (subscriptionRef.current) {
-            const [entities, clause] = await sdk.toriiQueryIntoHashedKeys(
-                fetchingRef.current!
-            );
-            await sdk.updateEntitySubscription(subscriptionRef.current, clause);
-            state.mergeEntities(entities);
-            return null;
-        }
+            if (subscriptionRef.current) {
+                const [results, clause] = await config.queryToHashedKeysMethod(
+                    fetchingRef.current!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)